### ISSUE TITLE
Search filters persist after tab changes

### ISSUE THREAD

**Issue by aquilent-dgemoets** [USDepartmentofLabor/Child-Labor-Android#54]
In both Goods and Country, if you enter a search filter via the magnifying glass, then change tabs and back again, the filter will still be applied even though there's no visual indication of it (other than a shorter list).  It's more serious if you have also filtered via the dropdowns.

The easiest work-around is to go back to the app's Main screen then select Goods (or Country) whereupon the search filter will be canceled.

**Comment by carolynhuang** [comment]
Please confirm whether this issue still exists.

**Comment by jfc3-dol** [comment]
When you type in "Brick" in the search on the "Goods" screen, then switch from the "A - Z" tab to the "By Sector" tab and back again the "Brick" search still is applied.

Maybe it's a feature and not an issue.

**Comment by jfc3-dol** [comment]
Same thing happens if you use the word "Silk" which has three items.

It seems to keep the prior search when switching tabs. Search for "brick" on one and "Silk" on the other. Then go back and forth and they will stay that way.

**Comment by carolynhuang** [comment]
Given the complexity of this issue and our limited remaining time for the final update, we will need to table this and see if a fix can be applied next year.

**Comment by jfc3-dol** [comment]
Okay, that works for me I guess.

At least it's documented so we can figure it out when it comes time to upgrade the application next year.

**Comment by NJICF** [comment]
Made the filter persist the search when moving between tabs.

**Comment by jfc3-dol** [comment]
When I did my testing on the goods page, I typed two different search terms, and they stayed with the tab they were on.

Under the "A-Z" tab I typed "brick", and two items showed in the list. On the "By Sector" tab I typed "bean", and three items showed in the list. When I tabbed back to the "A-Z" tab, the search term "brick" was still there.

The same concept applies to the "Countries" three tabs ("A-Z", "By Level', and "By Sector").

In the "A-Z" tab I typed "al "into the search INPUT file and got two results, while in the "By Level' tab I typed "be" and got two results. Finally, in the "By Sector" tab I typed "bur" and got three results. All these search terms stayed with there individual tabs.

Going to close the issue.
